### PR TITLE
Added auto-updating copyright year to homepage and to readme

### PR DESCRIPTION
**Pull request by specialk1st**
Year in index.gsp page will auto-update using javascript from now on.  Year in readme.md now reads 2013-2016 instead of 2013.

### CODE REVIEW

File: grails-app/views/index.gsp [hunk 1]
```diff
@@ -215,7 +215,7 @@
 
 <div class="footer">
     <div class="container">
-        <p><span class="glyphicon glyphicon-copyright-mark"></span> Gr8Ladies 2015</p>
+        <p><span class="glyphicon glyphicon-copyright-mark"></span> Gr8Ladies <script type="text/javascript">document.write(new Date().getFullYear());</script></p>
```
Comment (jlstrater) [inline grails-app/views/index.gsp]: Thank you so much for contributing. While you're in here, can you please update the text to be Gr8Ladies 2013 - <script... ?

Comment (specialk1st) [inline grails-app/views/index.gsp]: Sure Jenn!

**Comment by specialk1st** [comment]
The changes have been made.

**Comment by jlstrater** [comment]
Thanks again! :+1:

**Comment by specialk1st** [comment]
Thank you for spreading Grails awareness for us G8 ladies via your talks.  Hope to see you at G3 Summit this year!

**Pull request merged**